pkg/clusteragent/admission/patch/file_provider.go

```diff
@@ -43,48 +45,52 @@ func (fpp *filePatchProvider) subscribe(kind TargetObjKind) chan PatchRequest {
 }
 
 func (fpp *filePatchProvider) start(stopCh <-chan struct{}) {
+	log.Info("Starting file patch provider")
```

Comment:
:+1: thanks for this, was hard to tell before that it was working when I tried to configure it!